### ISSUE TITLE
Google Ads connector: custom queries configuration check fails because of automatic adding of 'segments.date' to custom queries 

### ISSUE THREAD

**Issue by anteakar** [airbytehq/airbyte#32546]
### Connector Name

Google Ads

### Connector Version

v2.0.3

### What step the error happened?

Configuring a new connector

### Relevant information

I'm sending Google Ads data to BigQuery. I'm self-hosting Airbyte. I wanted to create custom tables so I added Custom GAQL Queries. However some of my queries used tables that are not compatible with segments.date which is a field that is automatically added by Airbyte. Automatically adding this field makes it impossible to make custom queries using many Google Ads tables even though some default Airbyte tables use tables that are not compatible with segments.date. This feature results in me having to stop using Airbyte and go to another connector provider.

Example of a custom query I used: SELECT label.id, label.name, label.status, label.resource_name, label.text_label.background_color, label.text_label.description FROM label

There is a default Airbyte table identical to the one that would be made with the above query. I'm using this to illustrate the issue with this feature.

### Relevant log output

```shell
errors {
  error_code {
    query_error: PROHIBITED_SEGMENT_IN_SELECT_OR_WHERE_CLAUSE
  }
  message: "Cannot select or filter on the following segments: \'segments.date\'(could not support requested resources: \'LABEL\'), since segment is incompatible with the resource in the FROM clause or other selected segmenting resources."
}
```

### Contribute

- [ ] Yes, I want to contribute

**Comment by tolik0** [comment]
@anteakar Hi, this issue is fixed in the 3.1.0 version.

**Comment by lazebnyi** [comment]
fixed in the 3.1.0 version
